src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+@import "../variables/variables";
+
+.calendar {
+  margin: 100px auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 710px;
```

Comment:
Avoid such a hard-coded values.
This is like a magic number, it is not clear why is it set to 710px and if you change for instance dimensions of calendar-day things will break up.
You should calculate the width basing on your variables ;)